part-spec/power.json

```diff
@@ -697,6 +697,507 @@
+          "description": "minimum accuracy of battery charge voltage regulated by a device",
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryChargeVoltageAccuracyMax": {
+          "description": "maximum accuracy of battery charge voltage regulated by a device",
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "efficiency": {
+          "description": "charge efficiency vs charge current of a device",
+          "$ref": "graph.json#/graphDefiniton"
+        },
+        "vinTyp": {
+          "description": "typical input voltage under which a device can be expected to reliabily operate",
+          "comment": "units of volts",
+          "$ref": "definitions.json#/unit"
+        },
+        "vinMin": {
+          "description": "minimum input voltage under which a device can be expected to reliabily operate",
+          "comment": "units of volts",
+          "$ref": "definitions.json#/unit"
+        },
+        "vinMax": {
+          "description": "maximum input voltage under which the part can be expected to reliabily operate ",
+          "comment": "units of volts",
+          "$ref": "definitions.json#/unit"
+        },
+        "fswTyp": {
+          "description": "typical switching frequency of a device",
+          "comment": "units of hertz",
+          "$ref": "definitions.json#/unit"
+        },
+        "fswMin": {
+          "description": "minimum switching frequency of a device",
+          "comment": "units of hertz",
+          "$ref": "definitions.json#/unit"
+        },
+        "fswMax": {
+          "description": "maximum switching frequency of a device",
+          "comment": "units of hertz",
+          "$ref": "definitions.json#/unit"
+        },
+        "currentConsumption": {
+          "description": "current consumption of a device",
+          "type": "array",
+          "items": {
+            "$ref": "definitions.json#/currentConsumption"
+          }
+        },
+        "componentProtectionThresholds": {
+          "description": "Thermal and power supply protection thresholds of a device",
+          "type": "array",
+          "items": {
+            "$ref": "icProtection.json#/componentProtectionThresholds"
+          }
+        },
+        "integratedLoadSwitch": {
+          "description": "whether the device contains integrated power path load switch(es)",
+          "type": "boolean"
+        },
+        "integratedFets": {
+          "description": "whether the device contains integrated switching mosfets",
+          "type": "boolean"
+        },
+        "integratedFetProperties": {
+          "description": "describes integrated fet current limits and rdson properties",
+          "$ref": "#/$defs/powerFetProperities"
+        },
+        "BC12Support": {
+          "description": "whether bc 1.2 detection is built in",
+          "type": "boolean"
+        },
+        "TCPCSupport": {
```

Comment:
it's a bit weird that `usb` (and `pd` below) aren't capitalized but `TCPC` (and `BC` above) is. Seems like we should be consistent.

Reply:
Agreed. I'll make the change.